### PR TITLE
Fix the documentation about default themes

### LINKED ISSUE #17130: Theme & Provider folder empty in KeyCloak 20.0.3
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

docs

### Describe the bug

The documentation and nearly all online tutorials advice to take an existing theme in the subfolder "theme" and extend it to specific needs. The theme folder however is empty and there is absolutely NO TEMPLATE to start from. I don't even know where and how to start. Seriously guys: It would be quite more helpful to work on the documentation instead of adding features to keycloak. It has cost my company a week of work alone just to get keycloak running together with an old JSF1.5 App because nothing is documented correctly.

### Version

20.0.3

### Expected behavior

Well documented

### Actual behavior

Very poorly documented

### How to Reproduce?

Try to theme your keycloak login with the help of the keycloak documentation and the current server download (v 20.0.3)

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by mposolda**
closes #17130

### CODE REVIEW

File: docs/documentation/server_development/topics/themes.adoc [hunk 1]
```diff
@@ -41,10 +41,12 @@ NOTE: To set the theme for the `master` Admin Console you need to set the Admin
...
 ----
 
+[[_default-themes]]
 === Default themes
 
-{project_name} comes bundled with default themes in the server's root `themes` directory. To simplify upgrading you should not edit the bundled themes
-directly. Instead create your own theme that extends one of the bundled themes.
+{project_name} comes bundled with default themes in the JAR file `keycloak-themes-{project_versionMvn}.jar` inside the server distribution.
+The server's root `themes` directory does not contain any themes by default, but it contains README file with some additional details about the default themes.
+To simplify upgrading you should not edit the bundled themes directly. Instead create your own theme that extends one of the bundled themes.
```
Comment (andymunro) [inline docs/documentation/server_development/topics/themes.adoc:49]:
```suggestion
The server's root `themes` directory does not contain any themes by default, but it contains a README file with some additional details about the default themes.
To simplify upgrading, do not edit the bundled themes directly. Instead create your own theme that extends one of the bundled themes.
```

File: docs/documentation/server_development/topics/themes.adoc [hunk 2]
```diff
@@ -407,3 +409,8 @@ A single archive can contain multiple themes and each theme can support one or m
 
 To deploy the archive to {project_name}, add it to the `providers/` directory of
 {project_name} and restart the server if it is already running.
+
+[role="_additional-resources"]
+=== Additional resources for Themes
+* For the inspiration, see <<_default-themes,Default Themes>> bundled inside {project_name}.
+* {quickstartRepo_link}[{quickstartRepo_name}] - Directory `extension` of the quickstarts repository contains some theme examples, which can be also used as an inspiration
```
Comment (andymunro) [inline docs/documentation/server_development/topics/themes.adoc:416]:
```suggestion
* {quickstartRepo_link}[{quickstartRepo_name}] - Directory `extension` of the quickstarts repository contains some theme examples, which can be also used as an inspiration.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
@mposolda A few minor changes. Otherwise, it is approved.

**Comment by mposolda** [comment]
@andymunro Thanks!

**Review by andymunro — APPROVED** [review]

**Pull request merged**